### DISCUSSION TITLE
[Question] Cleanup of the Debian platform

### DISCUSSION THREAD

**Discussion by nicolasdonato** [lae/ansible-role-proxmox#157] [Q&A]
Hello,

The difference between the installation of Proxmox by your Ansible role or by the Proxmox ISO.
I am a beginner concerning Proxmox.
I know the ISO is built with some specifications and Proxmox has directly the capability to interact with the CPU, RAM, ...
I wish to know if your role will have the capability to clean the OS, change the configuration of different files, ... to allow to Proxmox installation to interact with the CPU direclty.

I thank you in advance.

**Comment by lae** [comment]
I'm not sure of the differences between the role and the ISO, but the role mostly just follows standard installation procedure for barebones Debian hosts, and I don't really have any reason to believe there's much of a difference between the ISO and barebones, other than a different installer.

This role does not clean the operating system in any special way. Standard installation procedure does involve installing Proxmox' kernel packages, which is what would interface with the CPU, so this role does install that kernel - and if I understand you correctly, that is probably what you're referring to. This role does not modify any system configuration files other than the SSH configuration file (in order to allow clustering to be automated) and any other files specified by PVE installation instructions/the role variables you configure for the role (read through the README for more information).

Please also consider asking your question on the Proxmox user forums so a Proxmox developer can pitch in with their advice as well.
